server/src/routers/parkingRouter.js

```diff
@@ -30,6 +31,16 @@ router.post('/parking/newAd/:userId', validateIds, async (req, res, next) => {
     })
     try {
         const ad = await newAd.save()
+        //Error handling required as in if anything is missing like longitude or latitude
```

Comment:
What does the validateIds middleware do?

Reply:
It checks the type of the Ids coming in the query. Although it can be considered redundant, but I had this issue when I was testing and sometimes the id was messed up and the route was behaving weirdly. And I thought I just keep it. It can eliminated on choice.